R/aggregate_sex.R

```diff
@@ -0,0 +1,66 @@
+#' Aggregate sex-specific counts to both sexes combined
+#'
+#' @param dt data.table with data to be aggregated.
+#' * must only contain columns specified in `id_cols` and `value_cols`.
+#' * must include a column called 'sex', and only include observations for
+#' 'female' and 'male' but not 'both'.
+#' * each combination of `id_cols` (not including 'sex') must include a row for 'female' and 'male'.
+#' * each combination of `id_cols` must uniquely identify each row.
+#' @param id_cols character vector of id columns that uniquely identify each row
```

Comment:
we should create a template or something so that we are consistent with these parameter descriptions (example: everywhere where we have an id_cols parameter we have this same description)

Reply:
You can inherit parameter or sections from functions in the same or different R package. See `Inheriting documentation from other topics` section https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html

We can use `demUtils` as the main package and inherit universal documentation to the other packages?